src/service/chat.js

```diff
@@ -206,7 +206,7 @@ const moveChatToAnon = async ({
   }
 };
 
-const initChatFromPost = async ({source, id}) => {
+const initChatFromPost = async ({source, id, postId}) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function signature of `initChatFromPost` has been changed to include a new parameter `postId`. Ensure that all calls to this function have been updated to pass the `postId` argument. If not, it could lead to unexpected behavior or errors.

```diff
- const initChatFromPost = async ({source, id}) => {
+ const initChatFromPost = async ({source, id, postId}) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->